src/main.rs

```diff
@@ -93,6 +94,12 @@ enum Command {
         #[clap(long)]
         bribe: bool,
     },
+
+    /// Class ascension command
+    #[clap(alias = "a", display_order = 5)]
```

Comment:
can we call this just Class, e.g. `rpg class rogue`? ascend sounds too mysterious

Reply:
I was thinking that may class with a class change parameter in the future if we wanted to add something like that